frameworks/g-cloud-9/questions/services/scalingType.yml

```diff
@@ -12,8 +12,10 @@ type: checkboxes
 options:
   - label: Scales automatically
     value: automatic
+    filter_label: automatic
   - label: Scales with user intervention
     value: user_intervention
+    filter_label: manual through user interface
```

Comment:
Does this fully encapsulate the answer suppliers might have selected? Not sure if 'user intervention' might also be a phone call, which wouldn't quite make sense calling a 'user interface'?